.github/workflows/close-stale.yml

```diff
@@ -13,3 +13,12 @@ jobs:
     steps:
       - name: Close Stale Issues
         uses: actions/stale@v3.0.19
+        with:
+          days-before-stale: 60
+          days-before-close: 7
+          stale-issue-message: This issue had no activity for **2 months**, and will be closed in **one week** unless there is new activity. Cheers!
+          stale-issue-label: stale
+          exempt-issue-labels: priority/p0,priority/p1,priority/p2
```

Comment:
What's the rationale for exempting these? I'd like to have a priority assigned to every issue, which conflicts with this.

Reply:
In light of everything having priority, then I would stick to excluding p0. Though this should not be an issue as if it is a 0 it should be addressed faster...

Reply:
I'd personally like to know if we lose track of any issue, especially a p0, but SGTM.